app/src/main/java/org/mozilla/rocket/content/ecommerce/ui/DealFragment.kt

```diff
@@ -93,11 +96,25 @@ class DealFragment : Fragment() {
     private fun observeAction() {
         runwayViewModel.openRunway.observe(this, Observer { action ->
             context?.let {
-                startActivity(ContentTabActivity.getStartIntent(
-                    it,
-                    action.url,
-                    action.telemetryData.copy(vertical = TelemetryWrapper.Extra_Value.SHOPPING, versionId = dealViewModel.versionId)
-                ))
+                when (action.type) {
+                    1 -> {
```

Comment:
Should we define the constants for the magic numbers?

Reply:
Yes! my bad, code updated for constant definitions. Thanks!